Gruntfile.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
+var path = require('path');
+var config = require('config');
+
+let dataDirectory = config.get('database.data_directory');
+if (dataDirectory === '') {
+    dataDirectory = path.join(__dirname, './.oak');
```

Comment:
`./.oak` is a relative path. The default value (`''`) is a special case. Any other specified value for `database.data_directory` must be an absolute path.